### ISSUE TITLE
accessibility for ibm.docs home page/ Focus visibility

### ISSUE THREAD

**Issue by Stefan8412** [carbon-design-system/carbon-for-ibm-dotcom#9108]
### Description

by executing accessibility scan on https://www.ibm.com/docs/en i see that there is 82 'focus visible ' needs review checks. I check the page manually with tabing through all page components and they looks ok, keyboard focus meets visibility standard when tabbing. Can you confirm that enough for this page and no action is neeed , confirm its false-positive

![Screenshot 2022-07-19 at 13 42 06](https://user-images.githubusercontent.com/57751227/179741984-168b85a7-961c-47d8-a772-40ef35246247.png)
d

### Component(s) impacted

home page for ibm docs , https://www.ibm.com/docs/en 

### Browser

Chrome

### Carbon for IBM.com version

"@carbon/ibmdotcom-react": "1.32.1",

### Severity

Severity 4 = The problem is not visible to or noticeable to an average user. Affects minor functionality, no workaround needed.

### Application/website

https://www.ibm.com/docs/en 

### Package

@carbon/ibmdotcom-react

### CodeSandbox example

masthead, card section,language selector, footer

### Steps to reproduce the issue (if applicable)

1. run accessibility scan on https://www.ibm.com/docs/en 
2. check needs review Focus visible issues 86 counted
3. tab with keyboard with through all the components on home page
4. confirm its false-positive

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/main/.github/CODE_OF_CONDUCT.md)
- [X] I checked the [current issues](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/issues) for duplicate issues

**Comment by oliviaflory** [comment]
Hi @Stefan8412 from the screen shot you shared the yellow icon is only a Warning, indicating issues that may not be a violation; and a manual review is needed. 

If you have performed a manual check to confirm that the Focus is highly visible, and the elements are tab-able and visible on keyboard focus as the checker suggested, your app is good!

**Comment by kennylam** [comment]
Confirmed that the warnings are not in violation.
